lua/lir/actions.lua

```diff
@@ -168,11 +168,11 @@ function actions.rename()
 end
 
 --- delete
-function actions.delete()
+function actions.delete(force)
```

Comment:
Should I document the `force` parameter as well?

Reply:
Yes, please document `force` as well.